bindings/c/test/mako/mako.cpp

```diff
@@ -321,9 +321,14 @@ int populate(Database db,
 		const auto key_begin = insertBegin(args.rows, worker_id, thread_id, args.num_processes, args.num_threads);
 		const auto key_end = insertEnd(args.rows, worker_id, thread_id, args.num_processes, args.num_threads);
 		auto key_checkpoint = key_begin; // in case of commit failure, restart from this key
+		double required_keys = (key_end - key_begin + 1) * args.load_factor;
 		for (auto i = key_begin; i <= key_end; i++) {
+			double r = rand() / (1.0 + RAND_MAX);
+			if (r > required_keys / (key_end - i + 1)) continue;
```

Comment:
Although `if() statement;` is syntactically correct, it is not very clear to read for someone new to the code.  It's much harder to see flow control at a glance vs
```
if (..) {
    statement;
}
```
or even
```
if (..)
    statement;
```
but the latter is more prone to error when adding temporary statements for debugging.